server/src/main/java/org/elasticsearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -711,6 +711,6 @@ public int getNumberOfInFlightFetches() {
      */
     public record CommandsResult(
         RoutingExplanations explanations, // Explanation for the reroute actions
-        ClusterState clusterState         // Resulting cluster state
+        ClusterState clusterState         // Resulting cluster state, should be removed in 9
```

Comment:
Similarly here, this'll have to wait until v10 for full removal. Could we mention a v8 symbol in the Javadoc tho?